### ISSUE TITLE
Cannot turn off analytics

### ISSUE THREAD

**Issue by JanKoppe** [zabbix/zabbix-community-docker#52]
I am unable to turn off the analytics via the environment variable, `XXL_analytics=false`. The container fails with the log

```
$ sudo docker logs -f zabbix-server
Nginx status page: allowed address set to 127.0.0.1.
PHP-FPM status page: allowed address set to 127.0.0.1.
[LOG 20:35:07] Preparing XXL features
sed: -e expression #1, char 273: unknown command: `/'
```

I'm also really not sure, why they are turned on by default. This should be made way more prominent, if not turned off or removed entirely.

**Comment by jangaraj** [comment]
Thx Jan for your report. Please follow #51, it's should be already fixed in dev image.

**Comment by JanKoppe** [comment]
Thanks, didn't see that issue. It still leaves my question regarding the default settings open. This should concern more users, who maybe haven't noticed it.

**Comment by jangaraj** [comment]
If I use opt-in concept, then statistic dataset will be very small to get any reasonable knowledge. For example I've seen significant number of user from Japan, so I've added support for japanese font. It was win-win situation for me and also for users. Also free is not free - see Facebook, Google, .... The analytics provide useful insights mainly for R&D. 

I work also for enteprise customers, so I know that any analytics collection can be a problem. Hopefully, infosec will stop project in that case. It was also reason for `XXL_analytics=false` switch. I've tried to add it transparently - it's in the readme + anybody can review code.
Feel free to create PR, if you know how to communicate it to users better. I'll be happy to review your PR.

Thanks Jan.
